code/mob/living/silicon/ai.dm

```diff
@@ -96,6 +96,8 @@ var/list/ai_emotions = list("Happy" = "ai_happy",\
 	var/mob/dead/aieye/eyecam = null
 
 	var/deployed_to_eyecam = 0
+	var/list/holograms = list()
```

Comment:
I think preference is to initialize empty lists in `New`? @ZeWaka

Reply:
That is correct. I forgot about that. But AI's are created so infrequently and mob/living/silicon/ai/New() is such a mess that I thought it would be better to just do it here.